### ISSUE TITLE
API call to get password status

### ISSUE THREAD

**Issue by aapeliv** [Couchers-org/couchers#618]
i.e. get whether the current user has a password or not.

**Comment by aapeliv** [comment]
@darrenvong the API will look something like this:

```proto3

service Account {
  rpc GetAccountInfo(google.protobuf.Empty) returns (GetAccountInfoRes) {
    // Get information about the user's account
  }

  // ...
}

message GetAccountInfoRes {
  enum LoginMethod {
    MAGIC_LINK = 0;
    PASSWORD = 1;
  }

  LoginMethod login_method = 1;
  bool has_password = 2;

  // future extensions, e.g. bool account_public = 3;
}
```

Sorry, I'm busy today and tomorrow morning with work, but will try to get this done asap.

**Comment by darrenvong** [comment]
No worries, it shouldn't block what I'm doing too much since the error message will still tell the user they don't have a password if they forget and enter one whilst trying to change password, so it's more of a user experience thing rather than it completely not working as it is.

**Comment by aapeliv** [comment]
Closed by #619
